optaplanner-docs/src/main/asciidoc/ConstraintStreams/ConstraintStreams-chapter.adoc

```diff
@@ -574,6 +574,121 @@ the stream.
 ====
 
 
+[[constraintStreamsTesting]]
+== Testing a constraint stream
+
+Constraint streams come with a unit testing harness, called "Constraint Verifier".
```

Comment:
Don't use ". Use _ for emphasis or * for bold.

Reply:
Suggestion:
Constraint streams include the Constraint Verifier unit testing harness.